### PR TITLE
start ipfs daemon for go-test workflow

### PR DESCRIPTION
**Pull request by galargh**
Syncing to commit https://github.com/protocol/.github/commit/2aab7c039d804a6fcc9d4b330ef23e7788b4291f.

### CODE REVIEW

File: cmd/git-remote-ipld/ipld_test.go [hunk 1]
```diff
@@ -1,15 +1,19 @@
+//go:build !windows || !386
+// +build !windows !386
```
Comment (marten-seemann) [inline cmd/git-remote-ipld/ipld_test.go:2]: Our stack _should_ build on 32 bit. Why is this necessary?

If there's a good reason, it would be good if that could be documented in a comment here.

Comment (galargh) [reply cmd/git-remote-ipld/ipld_test.go:2]: This is the comment I based the change on - https://github.com/ipfs-shipyard/git-remote-ipld/pull/35#discussion_r716137959

@aschmahmann do you have any more insight into this by any chance?

Comment (galargh) [reply cmd/git-remote-ipld/ipld_test.go:2]: I added a comment here: https://github.com/ipfs-shipyard/git-remote-ipld/pull/39/commits/d4c93371ed623dfa5c05059c41f353f16c58480b

**Comment by galargh** [comment]
I moved the changes introduced in this PR to https://github.com/ipfs-shipyard/git-remote-ipld/pull/41 and I intend to merge it to the Unified CI sync PR.

The win32 tests are disabled - the reason for it is included as a code comment.
